R/match-standards-chunks.R

```diff
@@ -79,8 +79,12 @@ hmdb_pos <- hmdb_neg <- Spectra(cdb)
 hmdb_pos$precursorMz <- mass2mz(hmdb_pos$exactmass, adduct = "[M+H]+")[, 1L]
 hmdb_neg$precursorMz <- mass2mz(hmdb_pos$exactmass, adduct = "[M-H]-")[, 1L]
 #' Neutral loss spectra
-hmdb_pos_nl <- neutralLoss(hmdb_pos, param = PrecursorMzParam())
-hmdb_neg_nl <- neutralLoss(hmdb_neg, param = PrecursorMzParam())
+nl_param <- PrecursorMzParam(filterPeaks = "removePrecursor", ppm = 50,
```

Comment:
@andreavicini , to explain: I remove any peaks matching the precursor m/z from the spectra before calculating the neutral loss. Reason: since we're subtracting the precursor m/z from the m/z of each peak in a spectrum we would always get a peak with a m/z = 0 - and these peaks would then always match. This leads IMHO to false positive matches.